upgrade/patches/41_custom_field_required.sql

```diff
@@ -0,0 +1,3 @@
+-- Add field to allow custom fields to be marked as required when editing
+alter table {{%custom_field}}
+    add `fld_edit_form_required` tinyint(1) NOT NULL DEFAULT 0;
```

Comment:
I've updated the patch to insert that column at the correct location.
However, I didn't have time to re-order all the columns.

For a very small table like custom_field I don't see this as an issue. It
would be more important for larger tables (like issue_custom_field). Also,
custom_field will pretty much always be looked up by PK, not by one of the
later columns.

On Mon, Nov 16, 2015 at 4:45 PM, Elan Ruusamäe <notifications@github.com>
wrote:

> In upgrade/patches/41_custom_field_required.sql
> <https://github.com/eventum/eventum/pull/107#discussion_r44996307>:
>
> > @@ -0,0 +1,3 @@
> > +-- Add field to allow custom fields to be marked as required when editing
> > +alter table {{%custom_field}}
> > +    add `fld_edit_form_required` tinyint(1) NOT NULL DEFAULT 0;
>
> please add AFTER statement here, so fields in table would be ordered like:
>
>    - primary key
>    - int values
>    - text values
>
> this supposedly improves key lookup performance.
> i'd like to follow this idea in eventum
>
> also probably should also order int fields too, but not sure in what
> order, smallest size first or biggest size first. or doesn't matter, just
> be consistent?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/eventum/eventum/pull/107/files#r44996307>.
>



-- 
Bryan Alsdorf
MariaDB Corporation | http://mariadb.com